modules/ycc_vm.py

```diff
@@ -881,10 +883,10 @@ def _get_resource_spec(memory, cores, core_fraction):
     return ResourcesSpec(memory=memory, cores=cores, core_fraction=core_fraction)
 
 
-def _get_network_interface_spec(subnet_id, assign_public_ip):
+def _get_network_interface_spec(subnet_id, assign_public_ip, assign_internal_ip):
```

Comment:
_is_same function should also check that if `assign_internal_ip` is set then ip address is same.

Update module docstring.